ios-deploy.c

```diff
@@ -1554,7 +1549,7 @@ void handle_device(AMDeviceRef device) {
         } else if (strcmp("rm", command) == 0) {
             remove_path(device);
         } else if (strcmp("exists", command) == 0) {
-            exit(app_exists(device));
+            app_exists(device);
```

Comment:
This removes ability to easily check app's existence from scripts without parsing stdout.